bot/exts/utilities/twemoji.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import re
+from typing import List, Literal
```

Comment:
As mentioned on discord, since a spring-cleanup-like PR (I forget the number), we use `list[]` etc. in place of the typing variant.